src/hip/hip_build_utils.cpp

```diff
@@ -39,6 +39,18 @@
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_VERBOSE)
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_DUMP)
 
+#ifdef MIOPEN_COMPILER_FLAGS_V2
+ /* Directory Path from ROCM PATH to MIOPEN Flag Path */
+char* MIOPEN_CMP_FLAG_REL_PATHS[MIOPEN_CMP_MAX_Flag] = { "llvm/bin/clang",
+                                                         "bin/clang-ocl",
+                                                         "llvm/bin/clang++",
+                                                         "llvm/bin/clang-offload-bundler",
+                                                         "\0" };
```

Comment:
Is this missing an entry for MIOPEN_ROCM_INSTALL_PATH to have appended?

Reply:
Updated the list, since MIOPEN_ROCM_INSTALL_PATH was intentionally mapped to \0 to return the base path only like a helper functionality for future use, but since it is not mandatory to provide this flag, to avoid complexity removed this for now. (Kept only the flags were hardcoding to be moved)